pkg/kubelet/container/runtime.go

```diff
@@ -470,6 +470,9 @@ type VolumeInfo struct {
 	// Whether the volume permission is set to read-only or not
 	// This value is passed from volume.spec
 	ReadOnly bool
+	// Inner volume name, which is the PV name if used, otherwise
+	// it is the same as the outer volume name.
```

Comment:
Maybe change outer volume name to "OuterVolumeSpecName" in the comment for consistency.

Reply:
done